package.json

```diff
@@ -27,6 +27,7 @@
   },
   "devDependencies": {
     "chai": "^4.1.2",
+    "eslint": "^5.6.0",
```

Comment:
If you add ESlint as a devDep you do not need to have it install on your PC to use it. It will get installed locally once you do `npm install`